Pearson-Advance/openedx-themes
# PADV-566: Add default Theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-566

## Description

This PR adds assets for default theme on edx-platform.

## Testing:

- Enable comprehensive theming.
- Run the LMS: `make dev.up.lms`.
- Set `pearson-theme` on a site.
- Compile the dafault theme.
- Verify that the default theme is applied correctly on Olive.

(For more information enter here: [How to enable themes)](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html#enable-themes)

## Screenshots

Log In:
![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/6979f20e-a089-425c-8772-c9e2cbd1792b)






Dashboard:
![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/fa7a33fb-d8f4-4bc4-bf10-2edc306c5184)





## Reviewers
